src/backend/web/graphql/index.js

```diff
@@ -0,0 +1,109 @@
+const { gql } = require('apollo-server-express');
```

Comment:
`graphql` would make this more readable

Reply:
I was trying to follow their own syntax, check [here](https://www.apollographql.com/docs/apollo-server/api/apollo-server/#gql).
I changed it to `graphql`.